trinity/algorithm/advantage_fn/rec_advantage.py

```diff
@@ -0,0 +1,184 @@
+"""REC advantage computation
```

Comment:
Only need to keep of of `rec_verl` and `rec`.
According to the configuration in `RECAlgorithm`, it is better to keep `rec` and remove `rec_verl`.